bot/resources/tags/ytdl.md

```diff
@@ -0,0 +1,8 @@
+Per [PyDis' Rule 5](https://pythondiscord.com/pages/rules), we are unable to assist with questions related to youtube-dl, commonly used by Discord bots to stream audio, as its use violates YouTube's Terms of Service.
+
+For reference, this usage is covered by the following clauses in [YouTube's TOS](https://www.youtube.com/static?template=terms), as of 2019-07-22:
+```
+The following restrictions apply to your use of the Service. You are not allowed to:
```

Comment:
Not sure why the language regarding redistribution was removed from this tag:

> 1. access, reproduce, download, distribute, transmit, broadcast, display, sell, license, alter, modify or otherwise use any part of the Service or any Content except: (a) as expressly authorized by the Service; or (b) with prior written permission from YouTube and, if applicable, the respective rights holders;
> 9. use the Service to view or listen to Content other than for personal, non-commercial use (for example, you may not publicly screen videos or stream music from the Service); or

Reply:
I removed it because it is just a side effect of youtube-dl, this tool in itself is only against 3rd paragraph, the first and 9th paragraph are about redistributibg the content created by tool such as youtube-dl, but aren't directly caused by it. For instance I could download my music collection using it and keep it on my computer, and it would only be against the 3rd paragraph.

Reply:
The overwhelming majority of use cases encountered on our server utilize ytdl to stream content with a discord bot, which is redistributing content outside of YouTube’s platform. That is why these sections were all included and why they should remain in the tag.

Please ask questions on these things before unilaterally making a decision to remove them.

Reply:
It just didn't made sense to me based on what was already in the tag.

Reply:
I've added 1 & 9.